container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend.go

```diff
@@ -229,7 +229,7 @@ func (backend *DockerKurtosisBackend) StartRegisteredUserServices(ctx context.Co
 
 	var restartPolicy docker_manager.RestartPolicy = docker_manager.NoRestart
 	if backend.productionMode {
-		restartPolicy = docker_manager.RestartOnFailure
+		restartPolicy = docker_manager.RestartAlways
```

Comment:
@h4ck3rk3y Do you know whey RestartOnFailure did not restart the container in DIVA use case?

Reply:
Yes, it didn't restart for DIVA. I built this image and restarted and DIVA was restartign!

Reply:
shouldn't they refactor their code to exit >0 if their service dies?

Reply:
So the service dying is the expected behavior in this case! There's an API call that you make and that API call is meant to restart the service but I empathize

Reply:
but probably yes, it's better to always restart a service in production mode